lib/google_scraping_web/controllers/api/v1/keyword_filter_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule GoogleScrapingWeb.Api.V1.KeywordFilterController do
```

Comment:
Generally we always use same index/list(`  get "/keywords", KeywordController, :index`) endpoint to do the filter based on params. Is it possible to use the same endpoint for filtering too?